website/docs/docs/dbt-extension-features.md

```diff
@@ -1,13 +1,14 @@
 ---
-title: dbt VS Code extension features and functionality
+title: dbt extension features
 id: dbt-extension-features
 description: "The dbt VS Code extension is backed by the dbt Fusion engine and a powerful LSP."
-sidebar_label: "dbt VS Code extension features"
+sidebar_label: "dbt extension features"
```

Comment:
I think from a branding perspective, it needs to say VS Code extension, if you're trying to shorten it, we could remove dbt
```suggestion
sidebar_label: "VS Code extension features"
```